deploy-addresses/blast.json

```diff
@@ -0,0 +1,6 @@
+{
+  "UniversalRouterV1": "0xe463635f6e73C1E595554C3ae216472D0fb929a9",
```

Comment:
```suggestion
  "UniversalRouterV1_2_NoV2Support": "0xe463635f6e73C1E595554C3ae216472D0fb929a9",
```

Reply:
ohh what's this one then? 0x643770E279d5D0733F21d6DC03A8efbABf3255B4